### PR TITLE
Create test/window directory and conftest file

### PR DESCRIPTION
**Pull request by mroeschke**
Precursor to https://github.com/twosigma/pandas/pull/2

### CODE REVIEW

File: pandas/tests/window/conftest.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+
+@pytest.fixture(params=["kaiser", "gaussian", "general_gaussian", "exponential"])
+def win_types_special(request) -> str:
+    return request.param
+
+
+@pytest.fixture(
+    params=["sum", "mean", "median", "max", "min", "var", "std", "kurt", "skew"]
+)
```
Comment (jreback) [inline pandas/tests/window/conftest.py:34]: can you add a 1-line comment on each of these

**Review by jreback — CHANGES_REQUESTED** [review]
I would do this versus pandas/master. Best to push whatever we can there in small uncontroversial bits.

**Comment by mroeschke** [comment]
Sounds good. I'll open this PR up against pandas/master instead.

**Comment by mroeschke** [comment]
Created https://github.com/pandas-dev/pandas/pull/27305 with the first cleanup

**Pull request closed without merging**